package.json

```diff
@@ -57,16 +57,16 @@
     "copy-webpack-plugin": "^6.0.3",
     "css-loader": "^5.0.2",
     "emotion-theming": "^10.0.14",
-    "eslint": "^8.3.0",
-    "eslint-config-airbnb": "^19.0.0",
-    "eslint-config-prettier": "^8.3.0",
+    "eslint": "~8.3.0",
+    "eslint-config-airbnb": "~19.0.0",
+    "eslint-config-prettier": "~4.1.0",
     "eslint-import-resolver-webpack": "^0.10.1",
-    "eslint-plugin-import": "^2.25.3",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-import": "~2.25.3",
+    "eslint-plugin-jsx-a11y": "~6.5.1",
     "eslint-plugin-node": "^9.1.0",
     "eslint-plugin-prettier": "^3.0.0",
-    "eslint-plugin-react": "^7.27.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
+    "eslint-plugin-react": "~7.28.0",
+    "eslint-plugin-react-hooks": "~4.3.0",
```

Comment:
How did you decide which packages to update and the versions to use?
Also, why not update `eslint` to 8.18.0?

Reply:
I almost asked the same question, but I suppose it doesn't really matter for eslint, right?

Reply:
ahhh, I wanted to continue using eslint-config-airbnb and found this in their documentation:
```
Install the correct versions of each package, which are listed by the command:
npm info "eslint-config-airbnb@latest" peerDependencies
```
so these were the versions provided to me by that command.

https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/README.md

Reply:
When I run that, I get
```
{
  eslint: '^7.32.0 || ^8.2.0',
  'eslint-plugin-import': '^2.25.3',
  'eslint-plugin-jsx-a11y': '^6.5.1',
  'eslint-plugin-react': '^7.28.0',
  'eslint-plugin-react-hooks': '^4.3.0'
}
```
which doesn't explain the downgrade of `eslint-config-prettier` or why you chose `eslint`  8.3.0

Reply:
It told me eslint 8.3.0, not 8.2.0